extensions/telemetry/main.ts

```diff
@@ -7,6 +7,7 @@ export default class TelemetryMainExtension extends LensMainExtension {
   async onActivate() {
     console.log("telemetry main extension activated")
     tracker.start()
+    tracker.reportPeriodically()
```

Comment:
previously reporting was done in both main and renderer processes, sending it from main processes only now.